buildrunner/docker/multiplatform_image_builder.py

```diff
@@ -333,17 +326,18 @@ def _build_single_image(self,
                 image_id = docker.image.inspect(tag_name).id
                 # Removes the image from host, if this fails it is considered a warning
                 try:
-                    docker.image.remove(images, force=True)
+                    logger.debug(f"Removing {tag_name}")
+                    docker.image.remove(tag_name, force=True)
                 except python_on_whales.exceptions.DockerException as err:
                     logger.warning(f"Failed to remove {images}: {err}")
             except python_on_whales.exceptions.DockerException as err:
                 logger.error(f"Failed to build {tag_name}: {err}")
                 raise err
 
-        built_images.append(ImageInfo(repo=name,
-                                      tags=tags,
-                                      platform=platform,
-                                      digest=image_id,))
+        self._intermediate_built_images[mp_image_name].append(ImageInfo(repo=name,
+                                                              tags=tags,
+                                                              platform=platform,
+                                                              digest=image_id,))
```

Comment:
IMHO, I think reading heavily indented lines is difficult and makes it harder to improve change things later on (i.e. adding another parameter, etc). I usually prefer this style:

```
        self._intermediate_built_images[mp_image_name].append(ImageInfo(
            repo=name,
            tags=tags,
            platform=platform,
            digest=image_id,
        ))
```

This is 100% my personal preference though, so I thought I'd bring it up as an alternative just in case you might like it better too? No worries on changing it if you prefer it this way though, seriously.

Reply:
I do like it better this way. I updated this with a new commit.